### ISSUE TITLE
safe_internet_search 관련 문의

### ISSUE THREAD

**Issue by kslim99** [swsnu/swppfall2020#26]
조교님 안녕하세요,

밑에 issue를 보고 safe_intrenet_search에서 에러 발생 시에 맨 첫 html 파일이 빈 파일이 되는 것은 run.py의 채점에 영향을 주지 않는다는 점은 이해했습니다. 그러면 이런 경우가 발생해도 fetch.py는 정상적으로 작동한 것으로 간주하면 될까요?

**Comment by hy00nc** [comment]
정확히 어떤 경우를 말씀하시는 지가 애매한데 어떤 에러를 말씀하시는 건가요?
이슈 #7 에서 말하는 에러인가요 아니면 임의로 테스트 했을 때 URLError가 발생하는 에러인가요 아니면 이슈 #12 에서 언급한 ranking data의 비일관성 문제인가요?

**Comment by kslim99** [comment]
이슈 #21 의 Q2에 언급된 문제입니다

**Comment by hy00nc** [comment]
네! 말씀하신대로 fetch.py를 실행하고 URLError가 발생했을 때 html이 빈 문서가 되는 현상은 정상이라고 생각하시면 됩니다.
해당 에러가 발생 한 뒤 html 상태를 채점 시 체크하진 않습니다.

**Comment by kslim99** [comment]
넵 감사합니다!
